src/round.js

```diff
@@ -18,10 +18,11 @@ class Round {
   }
 
   beginRound() {
-    this.currentTurn = new Turn(this.findPlayer(), this)
+    this.currentTurn = new Turn(this.players[0], this)
```

Comment:
Interesting that you are passing the instance of the round in the instantiation of the Turn. Why is this necessary? Could you do without this?